### PR TITLE
[daheinze-oscheer] hello

### LINKED ISSUE #5: [daheinze-oscheer] Hello World
Complete the Hello World lesson

### PR DESCRIPTION
**Pull request by oliverscheer**
Did the first tutorial https://quii.gitbook.io/learn-go-with-tests/go-fundamentals/hello-world

### CODE REVIEW

**Comment by oliverscheer** [comment]
godoc is not part of the devcontainer, could / should be added.

File: daheinze-oscheer/hello/hello.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package main
```
Comment (starlord-daniel) [inline daheinze-oscheer/hello/hello.go]: Package should have a meaningful name and description, like **package hello**

```suggestion
// Package hello contains code to send a nice greeting in some languages
package hello
```

File: daheinze-oscheer/hello/hello.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import "fmt"
+
+const spanish = "Spanish"
+const french = "French"
+const englishHelloPrefix = "Hello, "
+const spanishHelloPrefix = "Hola, "
+const frenchHelloPrefix = "Bonjour, "
+
+func Hello(name string, language string) string {
```
Comment (starlord-daniel) [inline daheinze-oscheer/hello/hello.go:13]: Each function should have a comment in the form of: <funcname> <verb> ....

```suggestion
// Hello returns a Hello World greeting for multiple languages
func Hello(name string, language string) string {
```

File: daheinze-oscheer/hello/hello_test.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		got := Hello("Elodie", "Spanish")
+		want := "Hola, Elodie"
+		assertCorrectMessage(t, got, want)
+	})
+
+	t.Run("in French", func(t *testing.T) {
+		got := Hello("Francis", "French")
+		want := "Bonjour, Francis"
+		assertCorrectMessage(t, got, want)
+	})
```
Comment (starlord-daniel) [inline daheinze-oscheer/hello/hello_test.go:36]: We can apply a for loop here to save some space and make it cleaner. Let's talk about this in our session later

**Review by starlord-daniel — CHANGES_REQUESTED** [review]

**Review by starlord-daniel — APPROVED** [review]

**Comment by oliverscheer** [comment]
followed tutorial hello

**Pull request merged**